### PR TITLE
fix: ensure peggo version cmd works correctly

### PR DESCRIPTION
**Pull request by alexanderbez**
closes: #207

### CODE REVIEW

**Review by facundomedica — APPROVED** [review]

**Comment by alexanderbez** [comment]
Do not merge until changelog is updated on `main`

**Comment by facundomedica** [comment]
> Do not merge until changelog is updated on `main`

Whoops! Sorry, I thought you meant to update it *after* this PR gets merged

**Pull request merged**
